city_scrapers/spiders/il_finance_authority.py

```diff
@@ -0,0 +1,171 @@
+import re
+from datetime import datetime
+from io import BytesIO, StringIO
+
+import scrapy
+from city_scrapers_core.constants import BOARD, COMMISSION, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text_to_fp
+from pdfminer.layout import LAParams
+from pdfminer.pdfparser import PDFSyntaxError
+
+
+class IlFinanceAuthoritySpider(CityScrapersSpider):
+    name = "il_finance_authority"
+    agency = "Illinois Finance Authority"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.il-fa.com/public-access/board-documents/"]
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Is there a reason for leaving this in? Right now it has no effect, but not sure if it was used earlier